libraries/chain/block_state.cpp

```diff
@@ -80,7 +105,7 @@ block_state::block_state(const block_header_state&                bhs,
                          const block_signing_authority&           valid_block_signing_authority,
                          const digest_type&                       action_mroot)
    : block_header_state(bhs)
-   , block(std::make_shared<signed_block>(signed_block_header{bhs.header}))
+   , block()
```

Comment:
No big deal, but why even have this line?

Reply:
Documentation. Makes it clear what the `std::move` is replacing below.